canvas_account_admin_tools/settings/local.py

```diff
@@ -43,22 +43,18 @@
             'course_short_title': 'Selenium Auto Test 101',
             'course_title': 'Selenium Automated Test Course 101',
             'template': 'None',  # No Template
-            'term': '1',
+            'term': 'string:5209',
```

Comment:
Indeed, tests were failing locally due to term 1 not found.  Updated to fix tests.  
<img width="427" alt="term_existing_course" src="https://cloud.githubusercontent.com/assets/2808493/20719299/7359a0a2-b629-11e6-9780-29d1f28cc0b7.png">